src/common/hotbar-store.ts

```diff
@@ -123,9 +125,14 @@ export class HotbarStore extends BaseStore<HotbarStoreModel> {
     }
   }
 
+  @action
   addToHotbar(item: CatalogEntityItem, cellIndex = -1) {
     const hotbar = this.getActive();
-    const newItem = { entity: { uid: item.id }};
+    const newItem = { entity: {
+      uid: item.id,
+      name: item.name,
```

Comment:
This change seems to have broken a migration in `src/migrations/hotbar-store/5.0.0-alpha.0.ts(25,9)`:
```
TS2739: Type '{ uid: any; }' is missing the following properties from type '{ uid: string; name: string; source: string; }': name, source
```